src/modules/components/asset/assetDataListItem/assetDataListItemStructure/assetDataListItemStructure.tsx

```diff
@@ -68,43 +40,26 @@ export const AssetDataListItemStructure: React.FC<IAssetDataListItemStructurePro
         fallback: '-',
     });
 
-    const formattedPriceChanged = formatterUtils.formatNumber(fiatAmountChanged, {
-        format: NumberFormat.FIAT_TOTAL_SHORT,
-        withSign: true,
-    });
-
-    const formattedPriceChangedPercentage = formatterUtils.formatNumber(priceChange / 100, {
-        format: NumberFormat.PERCENTAGE_LONG,
-        withSign: true,
-    });
-
     return (
-        <DataList.Item className={classNames('flex items-center gap-x-3 py-3 md:py-5', className)} {...otherProps}>
-            <Avatar src={logoSrc} responsiveSize={{ md: 'md', sm: 'sm' }} className="block" />
-            <div className="flex w-full min-w-0 shrink items-center justify-between gap-3">
-                <div className="flex flex-col gap-y-1 truncate">
-                    <span className="truncate text-base leading-tight text-neutral-800 md:text-lg">{name}</span>
-                    <p className="truncate text-sm leading-tight text-neutral-500 md:text-base">
-                        <span>{formattedAmount} </span>
-                        <span className="truncate">{symbol}</span>
-                    </p>
-                </div>
-                <div className="flex flex-col items-end justify-center gap-y-1">
-                    {fiatPrice ? (
-                        <>
-                            <span className="text-base leading-tight text-neutral-800 md:text-lg">
-                                {formattedPrice}
-                            </span>
-                            <div className="flex items-center gap-x-1">
-                                <span className={changedAmountClasses}>{formattedPriceChanged}</span>
-                                <Tag label={formattedPriceChangedPercentage!} variant={tagVariant} />
-                            </div>
-                        </>
-                    ) : (
-                        <span className="text-sm leading-tight text-neutral-800 md:text-base">
-                            {copy.assetDataListItemStructure.unknown}
-                        </span>
-                    )}
+        <DataList.Item
+            className={classNames('flex items-center justify-between gap-x-3 py-3 md:py-5', className)}
+            {...otherProps}
+        >
```

Comment:
Looks good and technically unrelated, but could we adjust the min-h on the skeleton down to 88.5 to match layout? The height on the tag was giving us a few extra pixels to compensate for.